tests/visual/virtual-list.html

```diff
@@ -17,78 +17,81 @@
             position: relative;
             overflow: visible;
         }
+        .upper {
+            text-transform: uppercase;
+        }
     </style>
 </head>
 
 <body>
 
     <div id="test-area" class="grid">
 
-        <kendo-popup class="k-animation-container k-animation-container-shown" dir="ltr" style="left: 8px; top: 38px; min-width: 174px; width: 174px; height: auto;">
+        <kendo-popup class="k-animation-container k-animation-container-shown" style="left: 8px; top: 38px; min-width: 174px; width: 174px; height: auto;">
```

Comment:
Clean up some inline styles from the test, so that the visual test image preview won't cut some of the height of the component.